connector.go

```diff
@@ -62,6 +63,15 @@ func (c *otConnector) Driver() driver.Driver {
 	return c.otDriver
 }
 
+func (c *otConnector) Close() error {
+	// database/sql uses a type assertion to check if connectors implement io.Closer.
+	// The type assertion does not pass through to otConnector.Connector, so we explicitly implement it here.
+	if closer, ok := c.Connector.(io.Closer); ok {
```

Comment:
Could you please add an implement assertion in line 26 to show `otConnector` implements `io.Closer`?
https://github.com/XSAM/otelsql/pull/199/files#diff-c1caa8de39e277339469495d65efd1a363c54065c3cc02e7422b097774e56c86R25
It should be

```go
var _ io.Closer = (*otConnector)(nil)
```